packages/react-components/src/components/chart/contextMenu/ChartContextMenu.tsx

```diff
@@ -23,12 +23,19 @@ const ChartContextMenu = ({
   onOutSideClickHandler,
   trendCursors,
 }: ChartContextMenu) => {
+  /**
+   * Echarts tooltip has z-index of 9999999.
+   * Must set context menu above this so its always displayed on top.
+   */
+  const toolTipZIndex = 9999999;
```

Comment:
This should be a constant.. i believe we have a dedicate file for these constants. 
Also, not sure if we need 2 variables here, the comments gives exellent context as to why the value is so high, so probably can do with just one vairable `MENU_Z_INDEX = 10000000`

Reply:
Sounds good, I updated to make it a constant